pkg/render/goldmane/component.go

```diff
@@ -123,23 +131,48 @@ func (c *Component) serviceAccount() *corev1.ServiceAccount {
 	}
 }
 
+// configMap returns a ConfigMap containing configuration for Goldmane that does not require a restart of the pod.
+// This is mounted as a file within the Pod, which can be dynamically reloaded when changed.
+func (c *Component) configMap() *corev1.ConfigMap {
+	type configMapData struct {
+		EmitFlows bool `json:"emitFlows"`
+	}
+
+	d, err := json.Marshal(configMapData{EmitFlows: c.cfg.ManagementClusterConnection != nil})
+	if err != nil {
+		logrus.WithError(err).Fatal("BUG: Failed to marshal config map data")
+	}
+
+	return &corev1.ConfigMap{
+		TypeMeta:   metav1.TypeMeta{Kind: "ConfigMap", APIVersion: "v1"},
+		ObjectMeta: metav1.ObjectMeta{Name: GoldmaneName, Namespace: GoldmaneNamespace},
+		Data: map[string]string{
+			GoldmaneConfigFileName: string(d),
+		},
+	}
+}
+
 func (c *Component) goldmaneContainer() corev1.Container {
+	guardianSvc := render.GuardianService(c.cfg.ClusterDomain)
 	env := []corev1.EnvVar{
 		{Name: "LOG_LEVEL", Value: "INFO"},
 		{Name: "PORT", Value: fmt.Sprintf("%d", GoldmaneServicePort)},
 		{Name: "SERVER_CERT_PATH", Value: c.cfg.GoldmaneServerKeyPair.VolumeMountCertificateFilePath()},
 		{Name: "SERVER_KEY_PATH", Value: c.cfg.GoldmaneServerKeyPair.VolumeMountKeyFilePath()},
 		{Name: "CA_CERT_PATH", Value: c.cfg.TrustedCertBundle.MountPath()},
+		{Name: "PUSH_URL", Value: fmt.Sprintf("%s/api/v1/flows/bulk", guardianSvc)},
```

Comment:
I know this isn't likely to change, but this also isn't something that needs to force a restart of goldmane so we could add it to the config map.

Reply:
We don't support changing this without a restart in Goldmane today - we could add that capability, but would require a small refactor and I don't think is something we really need at this point?

Reply:
No we don't need to do it, but this is configuring the client and you basically added the ability to reload the configuration for the client if it changes. So it seems less intuitive to me that some config changes would force a restart for the emitter and some won't.

I've no problem leaving this as is.

Reply:
I don't have a reason to implement this - it's not something we need. If it's easy to do, we can implement it if the need ever arises. Let's leave it as is.